src/test/java/it/gov/pagopa/pu/fileshare/service/FileStorerServiceTest.java

```diff
@@ -164,4 +171,38 @@ void givenInvalidPathWhenSaveToSharedFolderThenInvalidFileException() {
       }
     }
   }
+
+  @Test
+  void givenExistingFileWhenDecryptFileThenReturnInputStreamResource() throws IOException {
+    InputStream cipherInputStream = new ByteArrayInputStream("Decrypted content".getBytes());
```

Comment:
an InputStream should be closed after the usage, is it necessary to create it or could you use Mockito instead to create it?
```suggestion
    InputStream cipherInputStream = Mockito.mock(ByteArrayInputStream.class);
```